dev-python/configshell-fb/configshell-fb-1.1.29-r2.ebuild

```diff
@@ -39,6 +39,14 @@ PATCHES=(
 	"${DISTDIR}/${P}-replace-getargspec-2.patch"
 )
 
+src_prepare() {
+	default
```

Comment:
```suggestion
	distutils-r1_src_prepare
```

Reply:
Thanks! I thought that `default` would call `distutils-r1_src_prepare` because `EXPORT_FUNCTIONS` would overlaod it but apparently this is not true.